scripts/summarize_lake_notaro.R

```diff
@@ -23,16 +23,15 @@ library(lubridate)
 id <- 'nhd_13344210'
```

Comment:
I'd drop this file, it doesn't belong here as it's part of the modeling steps (so mda.lakes, where I already have a generalized version)